algorithm_catalog/eurac_pv_farm_detection.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "eurac_pv_farm_detection",
+  "type": "Feature",
+  "conformsTo": [
+    "http://www.opengis.net/spec/ogcapi-records-1/1.0/req/record-core"
+  ],
+  "properties": {
+    "created": "2024-10-16T00:00:00Z",
+    "updated": "2024-10-16T00:00:00Z",
+    "type": "apex_algorithm",
+    "title": "EURAC Photovoltaic Farms mapping",
+    "description": "A photovoltaic farm algorithm.",
+    "cost_estimate": "0.001",
+    "cost_unit": "platform credits per km\u00b2",
+    "keywords": [
+      "photovoltaic farm",
+      "detection"
+    ],
+    "language": {
+      "code": "en-US",
+      "name": "English (United States)"
+    },
+    "languages": [
+      {
+        "code": "en-US",
+        "name": "English (United States)"
+      }
+    ],
+    "contacts": [
+      {
+        "name": "Hans Vanrompay",
+        "position": "Researcher",
+        "organization": "VITO",
+        "links": [
+          {
+            "href": "https://www.vito.be/",
+            "rel": "about",
+            "type": "text/html"
+          },
+          {
+            "href": "https://github.com/HansVRP",
+            "rel": "about",
+            "type": "text/html"
+          }
+        ],
+        "contactInstructions": "Contact via VITO",
+        "roles": [
+          "principal investigator"
+        ]
+      },
+      {
+        "name": "VITO",
+        "links": [
+          {
+            "href": "https://www.vito.be/",
+            "rel": "about",
+            "type": "text/html"
+          }
+        ],
+        "contactInstructions": "SEE WEBSITE",
+        "roles": [
+          "processor"
+        ]
+      }
+    ],
+    "themes": [
+      {
+        "concepts": [
+          { "id": "Sentinel-2 MSI" }
+        ],
+        "scheme": "https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/sciencekeywords"
+      }
+    ],
+    "formats": [
+      "GeoTiff", "netCDF"
+    ],
+    "license": "CC-BY-4.0"
+  },
+  "linkTemplates": [],
+  "links": [
+    {
+      "rel": "openeo-process",
+      "type": "application/json",
+      "title": "openEO Process Definition",
+      "href": "https://raw.githubusercontent.com/ESA-APEx/apex_algorithms/refs/heads/hv_pv_detection/openeo_udp/eurac_pv_farm_detection/eurac_pv_farm_detection.json"
```

Comment:
Note that this reference will break once the `hv_pv_detection` feature branch is removed

also see https://github.com/ESA-APEx/apex_algorithms/issues/48